test1

```diff
@@ -1,2 +1,3 @@
 first file for this organization
 This is my first change
+defect 1 is fixed
```

Comment:
looks good

Reply:
but gives comments

Reply:
comment is given in code. please check once